.github/scripts/build_npm_applications.sh

```diff
@@ -0,0 +1,46 @@
+PRE_OR_POST_RELEASE=$1
+MODULE_TYPE=$2
+
+
+if [ "$PRE_OR_POST_RELEASE" = "PRE" ]; then 
+    if [ "$MODULE_TYPE" = "NPM-ES" ]; then  
+        cd smoke/smoke-test-application-NPM-ES
+        npm uninstall aws-rum-web
+        npm run clean 
+        npm install 
+        npm install /home/runner/work/aws-rum-web/aws-rum-web #Install locally 
+        npm run build
+    elif [ "$MODULE_TYPE" = "NPM-CJS" ]; then  
+        cd smoke/smoke-test-application-NPM-CJS
+        npm uninstall aws-rum-web
+        npm run clean 
+        npm install 
+        npm install $(npm pack /home/runner/work/aws-rum-web/aws-rum-web | tail -1) #Install locally 
```

Comment:
question: Why does this command differ from the `NPM-ES` build?

Reply:
This needs to be done as `npm install <path to local>` creates a symlink to the directory instead of actually copying the files over. When used with webpack and babel, if there is only a symlink, it's known to be problematic when the web bundle is created and the dependencies are resolved. The solution therefore is to use `npm pack ` which will copy over the files to the directory and install them. 

With the CJS build, I kept facing issues using the `npm install <path to local>` option. It was not  an issue with ES build (unclear to me why the difference exists). But therefore, there are two different commands being used. I believe the ES build will work with `npm pack` as well. 


Reference: 

https://stackoverflow.com/questions/51845893/error-importing-local-npm-package
https://stackoverflow.com/questions/21425980/npm-install-without-symlinks-option-not-working